resources/lang/en/events/lead_ceremony_df.json

```
@@ -395,7 +397,7 @@
       "stubbornness", "authority"]
         },
         {
-          "text": "With steady footing, r_c approaches m_c, {PRONOUN/r_c/poss} velvet paws treading noiselessly upon the moss-covered ground. Brooding demeanor, accompanied by an ominous silence, alludes to a dark past that remains locked within {PRONOUN/r_c/poss} enigmatic spirit. {PRONOUN/r_c/subject/CAP} {VERB/r_c/give/gives} a life for [virtue], then {VERB/r_c/leave/leaves} without a sound escaping {PRONOUN/r_c/poss} tongue.",
+          "text": "With steady footing, r_c approaches m_c, {PRONOUN/r_c/poss} velvet paws treading noiselessly upon the spongy ground. r_c's brooding silence and ominous silence allude to the dark past that remains locked within {PRONOUN/r_c/poss} enigmatic spirit. {PRONOUN/r_c/subject/CAP} silently {VERB/r_c/give/gives} a life for [virtue], then {VERB/r_c/leave/leaves} without ever making a sound.",
```

```suggestion
          "text": "With steady footing, r_c approaches m_c, {PRONOUN/r_c/poss} velvet paws treading noiselessly upon the spongy ground. r_c's brooding demeanor and ominous silence allude to the dark past that remains locked within {PRONOUN/r_c/poss} enigmatic spirit. {PRONOUN/r_c/subject/CAP} silently {VERB/r_c/give/gives} a life for [virtue], then {VERB/r_c/leave/leaves} without ever making a sound.",
```
I think that's the american spelling (demeanour but make it eagle noises)